fuse-box/fuse-box
# WIP: New website and documentation - Docosaurus

To test it out (very early stage):
```sh
cd website
npm install
npm run start
```

Please let me know what you think, we should agree on what to put on the main page and so on. There is an option to have a blog (maybe for breaking releases and so on?).
